tests/test_verify_build.py

```diff
@@ -267,3 +267,34 @@ def {}(session):
         assert (session / "Scripts" / "nox.exe").exists()
     else:
         assert (session / "bin" / "nox").exists()
+
+
+def test_pip_install_m2crypto(pipexec, build, tmpdir):
+    packages = [
+        "salt",
+    ]
+    env = os.environ.copy()
+    env["RELENV_DEBUG"] = "yes"
+    p = subprocess.run(
+        [str(pipexec), "install", "m2crypto", "--no-cache", "-v"],
+        env=env,
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE,
+    )
+    assert p.returncode == 0, f"Failed to pip install m2crypto"
+    gcc = str(
+        pathlib.Path(DATA_DIR)
+        / "toolchain"
+        / f"{get_triplet()}"
+        / "bin"
+        / f"{get_triplet()}-gcc"
+    )
+    include = str(pathlib.Path(build) / "include")
+    found_include = False
+    for _ in p.stderr.splitlines():
```

Comment:
https://pytest-shell-utilities.readthedocs.io/en/latest/

You could use this, and look at the last example shown.

Reply:
Noted.